pennylane/templates/layers.py

```diff
@@ -162,7 +164,7 @@ def RandomLayers(weights, wires, ratio_imprim=0.3, imprimitive=CNOT, rotations=N
                     seed=seed)
 
 
-def RandomLayer(weights, wires, ratio_imprim=0.3, imprimitive=CNOT, rotations=None, seed=None):
+def RandomLayer(weights, wires, ratio_imprim=0.3, imprimitive=CNOT, rotations=None, seed=42):
```

Comment:
Do we expect users to ever pass `None` as a value here? There's a line below (195|206) checking if `seed is not None` which may not serve its original purpose any more (i.e. we may want to raise an exception or let the seeding fail if `None` is passed.)

Reply:
There is still a use-case for setting seed to `None` (see the note below)